include/macros.h

```diff
@@ -11,7 +11,9 @@
 #define ALIGN16(val) (((val) + 0xF) & ~0xF)
 
 #define SQ(x) ((x)*(x))
+#define ROUND(x) (s32)(((x) >= 0.0) ? ((x) + 0.5) : ((x) - 0.5))
```

Comment:
This probably belongs in the libultra header for the system it's used in, since I don't think this was a macro commonly used.

Reply:
Added in gu,h. Would that be the right place for it?

Reply:
Seems fine yeah